ledger/src/blockstore.rs

```diff
@@ -3288,6 +3288,23 @@ fn send_signals(
     }
 }
 
+/**
+ * For each slot in the slot_meta_working_set which has any change, update
+ * its meta-data entry in the cf::SlotMeta.
+ *
+ * The column families which it may update are: cf::SlotMeta
```

Comment:
Another thought - technically we're updating the `write_batch` so maybe something like:

>The cf::SlotMeta column family may have an update added to `write_batch`